eng/common/scripts/update-docs-metadata.ps1

```diff
@@ -93,8 +93,7 @@ function GetAdjustedReadmeContent($pkgInfo, $lang){
 }
 
 $apiUrl = "https://api.github.com/repos/$repoId"
-$pkgs = VerifyPackages -pkgRepository $Repository `
-  -artifactLocation $ArtifactLocation `
+$pkgs = VerifyPackages -artifactLocation $ArtifactLocation `
```

Comment:
Can we clean-up the `$Repository` variable?

Reply:
What is your plan to clean-up some of these unused variables we have as a parameter to this script like `$Repository`?